Sources/Testing/Testing.docc/testing-for-errors-in-swift-code.md

```diff
@@ -0,0 +1,68 @@
+# Testing for errors in Swift code
+
+<!--
+This source file is part of the Swift.org open source project
+
+Copyright (c) 2024 Apple Inc. and the Swift project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+Ensure that your code handles errors in the way you expect.
+
+## Overview
+
+Write tests for your code that validate the conditions in which the
+code throws errors, and the conditions in which it returns without
+throwing an error. Use overrides of the ``expect(_:_:sourceLocation:)`` and
```

Comment:
```suggestion
throwing an error. Use overloads of the ``expect(_:_:sourceLocation:)`` and
```

Or "variants", etc. "Overrides" refers to a feature of inheritance which doesn't apply to macros.